vrc-get-gui/app/_main/packages/templates/index.tsx

```diff
@@ -436,24 +451,82 @@ function TemplateEditor({
 	dialog: DialogContext<boolean>;
 }) {
 	const [baseTemplate, setBaseTemplate] = useState<string>(
-		template?.base ?? "com.anatawa12.vrc-get.blank",
+		template?.base ?? "com.anatawa12.vrc-get.vrchat.avatars",
+	);
+	const [name, setName] = useState(template?.display_name ?? "New Template");
+	const [unityRange, setUnityRange] = useState(template?.unity_version ?? VRCSDK_UNITY_VERSIONS[0]);
+	const [nameTouched, setNameTouched] = useState(false);
+
+	// Package IDs for each base template
+	const AVATARS_PACKAGES = [
+		"com.vrchat.avatars",
+		"com.vrchat.base",
+		"com.vrchat.core.vpm-resolver",
+	];
+	const WORLDS_PACKAGES = [
+		"com.vrchat.worlds",
+		"com.vrchat.base",
+		"com.vrchat.core.vpm-resolver",
+	];
+
+	// State for selected packages (by package id)
+	const [selectedPackages, setSelectedPackages] = useState<string[]>(
+		template ? Object.keys(template.vpm_dependencies) : AVATARS_PACKAGES
+	);
+
+	// When baseTemplate changes, update selectedPackages accordingly
+	React.useEffect(() => {
+		if (baseTemplate === "com.anatawa12.vrc-get.vrchat.avatars") {
+			setSelectedPackages((prev) => {
+				// Remove worlds SDK if present, add avatars SDK
+				let next = prev.filter(
+					(id) => id !== "com.vrchat.worlds" && id !== "com.vrchat.avatars"
+				);
+				next = next.filter(
+					(id) => !AVATARS_PACKAGES.includes(id)
+				);
+				return [...next, ...AVATARS_PACKAGES];
+			});
+		} else if (baseTemplate === "com.anatawa12.vrc-get.vrchat.worlds") {
+			setSelectedPackages((prev) => {
+				// Remove avatars SDK if present, add worlds SDK
+				let next = prev.filter(
+					(id) => id !== "com.vrchat.avatars" && id !== "com.vrchat.worlds"
+				);
+				next = next.filter(
+					(id) => !WORLDS_PACKAGES.includes(id)
+				);
+				return [...next, ...WORLDS_PACKAGES];
+			});
+		} else if (baseTemplate === "com.anatawa12.vrc-get.blank") {
+			setSelectedPackages((prev) => prev.filter(
+				id =>
+				id !== "com.vrchat.avatars" &&
+				id !== "com.vrchat.worlds" &&
+				id !== "com.vrchat.base" &&
+				id !== "com.vrchat.core.vpm-resolver"
+			));
+		}
+	}, [baseTemplate]);
+
+	// Fetch available packages and repository info
+	const information = useSuspenseQuery(environmentProjectCreationInformation);
+	const packagesResult = useSuspenseQuery(environmentPackages);
+	const repositoriesInfo = useSuspenseQuery(environmentRepositoriesInfo);
+	const unityVersionsResult = useSuspenseQuery(environmentUnityVersions);
```

Comment:
use `useSuspenceQueries` to make them request parallelly.
When we use `useSuspenseQuery` in a row, it will be a request waterfall problem